failures "TypeError: <group.GOOD: np.uint32(0)> cannot be pickled"

Several devdeps jobs on main are failing with failures in the jump step tests:
https://github.com/spacetelescope/romancal/actions/runs/11799373327/job/32867751760

```
FAILED romancal/jump/tests/test_jump_step.py::test_one_CR[half] - TypeError: <group.GOOD: np.uint32(0)> cannot be pickled
FAILED romancal/jump/tests/test_jump_step.py::test_one_CR[all] - TypeError: <group.GOOD: np.uint32(0)> cannot be pickled
FAILED romancal/jump/tests/test_jump_step.py::test_two_CRs[half] - TypeError: <group.GOOD: np.uint32(0)> cannot be pickled
FAILED romancal/jump/tests/test_jump_step.py::test_two_CRs[all] - TypeError: <group.GOOD: np.uint32(0)> cannot be pickled
```

--- a/changes/425.bugfix.rst
+++ b/changes/425.bugfix.rst
@@ -1,0 +1,1 @@
+Fix Enum bug in python < 3.11 for the dqflags.

--- a/src/roman_datamodels/dqflags.py
+++ b/src/roman_datamodels/dqflags.py
@@ -19,14 +19,24 @@ which provides 32 bits. Bits of an integer are most easily referred to using
 the formula `2**bit_number` where `bit_number` is the 0-index bit of interest.
 """
 
-from enum import Enum, unique
+import sys
+from enum import unique
 
-import numpy as np
+# Something with pickling of multiclassed enums was changed in 3.11 + allowing
+# us to directly us `np.uint32` as the enum object rather than a python `int`.
+if sys.version_info < (3, 11):
+    from enum import IntEnum
+else:
+    from enum import Enum
+
+    import numpy as np
+
+    class IntEnum(np.uint32, Enum): ...
 
 
 # fmt: off
 @unique
-class pixel(np.uint32, Enum):
+class pixel(IntEnum):
     """Pixel-specific data quality flags"""
 
     GOOD             = 0      # No bits set, all is good
@@ -64,7 +74,7 @@ class pixel(np.uint32, Enum):
 
 
 @unique
-class group(np.uint32, Enum):
+class group(IntEnum):
     """Group-specific data quality flags
         Once groups are combined, these flags are equivalent to the pixel-specific flags.
     """

--- a/tests/test_dqflags.py
+++ b/tests/test_dqflags.py
@@ -1,3 +1,4 @@
+import sys
 from math import log10
 
 import numpy as np
@@ -31,7 +32,10 @@ def test_pixel_flags(flag):
     assert isinstance(flag, dqflags.pixel)
 
     # Test that the pixel flags are ints
-    assert isinstance(flag, np.uint32)
+    if sys.version_info < (3, 11):
+        assert isinstance(flag, int)
+    else:
+        assert isinstance(flag, np.uint32)
 
     # Test that the pixel flags are dict accessible
     assert dqflags.pixel[flag.name] is flag
@@ -79,7 +83,10 @@ def test_group_flags(flag):
     assert isinstance(flag, dqflags.group)
 
     # Test that the group flags are ints
-    assert isinstance(flag, np.uint32)
+    if sys.version_info < (3, 11):
+        assert isinstance(flag, int)
+    else:
+        assert isinstance(flag, np.uint32)
 
     # Test that the group flags are dict accessible
     assert dqflags.group[flag.name] is flag

